src/common/Browser.ts

```diff
@@ -330,6 +342,11 @@ export class Browser extends EventEmitter {
         ? this._contexts.get(browserContextId)
         : this._defaultContext;
 
+    const shouldIgnoreTarget = await this._targetFilterCallback(targetInfo);
```

Comment:
I think we should follow the JS language precedent of `Array.prototype.filter`, which works the other way around: if the callback returns `true`, it means the item is preserved. WDYT?

Reply:
perhaps it'd be better to call it `shouldAttachToTarget` or smth like this?

Reply:
Yes, that makes a lot of sense! I'll update the patch :)